src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -253,7 +253,7 @@ translation `p+L`, where `p` is only unique modulo `L`. The return type is a
 dict, the key `:base_points` gives an iterator over such `p`, and the key
 `:lineality_basis` lets one access a basis for the lineality space `L` of `PC`.
 
-See also [`vertices`](@ref) and [`lineality_space`](@ref).
+See also [`vertices(as::Type{PointVector{T}}, PC::PolyhedralComplex{T}) where {T<:scalar_types}`](@ref) and [`lineality_space`](@ref).
```

Comment:
```suggestion
See also [`vertices`](@ref vertices(as::Type{PointVector{T}}, PC::PolyhedralComplex{T}) where {T<:scalar_types}) and [`lineality_space`](@ref).
```
This should do the trick

Reply:
One moment, I'll give it a try on my machine (I remember it not working yesterday evening).

Reply:
Works like a charm, not sure what I did yesterday. :-/

Reply:
It took me a few tries of adding/removing `` ` `` before I found a working version.